indico/modules/events/util.py

```diff
@@ -385,7 +387,10 @@ def track_time_changes(auto_extend=False):
 
     :param auto_extend: Whether entry parents will get their end_dt
                         automatically extended or not.
+    :param user: The `User` that will trigger time changes.
```

Comment:
"The `User` used to perform access checks of cascades operations" or something similar would be more accurate